db/migrations/20230209071758_create_products_modification_1.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+ALTER TABLE products
+  ADD product_types_id INT,
+  ADD created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  ADD updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP;
```

Comment:
- 참조하는 외래키 컬럼명은 단수형으로 작성해주세요!
- 외래키에 대한 제약조건을 설정하는 코드를 추가해주세요

```suggestion
ALTER TABLE products
  ADD product_type_id INT,
  ADD created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  ADD updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP;
```